### ISSUE TITLE
Board page is not working

### ISSUE THREAD

**Issue by daksamit** [vcapretz/bull-board#12]
Before i've used to bull-arena package.
When i tried to use bull-board, nothing happens.
With no queues i have "no stats to display" page, but if I add some existing queue like 
```
import { createQueues } from 'bull-board'
export const myQueue = new Queue('MyQueue', myConfig)
```
And I try to use it, there is empty white page on '/admin/queues' and I have following errors in browser's console

```
TypeError: Expected a finite number, got number: NaN
    at e.exports (bundle.js:1)
    at $e (bundle.js:30)
    at Ai (bundle.js:22)
    at Vl (bundle.js:22)
    at Fl (bundle.js:22)
    at Ll (bundle.js:22)
    at Rl (bundle.js:22)
    at bundle.js:22
    at t.unstable_runWithPriority (bundle.js:30)
    at ca (bundle.js:22)
```

![image](https://user-images.githubusercontent.com/7165997/64493671-33049d80-d283-11e9-9055-6f8e340367a1.png)

**Comment by vcapretz** [comment]
Hey @daksamit thanks for reporting the issue! 

are you trying to create queues directly from Bull? that's not supported for now so you have to create the queues with the `createQueues` method, and then use then (at this point the API will be exactly as in Bull, since we don't modify it)

so a minimal code example would be something like: 
```js
const { createQueues, UI } = require('bull-board')

const redisConfig = {
  redis: {
    port: process.env.REDIS_PORT,
    host: process.env.REDIS_HOST,
    password: process.env.REDIS_PASSWORD,
  },
}

const queues = createQueues(redisConfig)
const queue = queues.add('myFirstQueue') // adds a queue

// defines how the queue works
queue.process(async job => {
  console.log(`Hello world`)
})

queue.add({ hello: 'world' })
``` 

I'll look into the bug tho, if you have queue data that shouldn't happen :D

**Comment by daksamit** [comment]
Yes, queues was created directly with bull, but I replaced it with `{ createQueues }` for compatibility.
(Sorry, I misstyped in my comment, I used bull-board api)
If I tried with only one queue to create using your api, error occurs anyway.
Maybe somewhere I have unnecessary configs in my app. but I decided to create issue, maybe it's a bug :)

**Comment by sjsn** [comment]
I was getting this bug too. For me the cause was that I'm using Heroku's redis-cloud add-on which does not expose `total_system_memory`.

So in the [`RedisStats.js`](https://github.com/vcapretz/bull-board/blob/master/ui/components/RedisStats.js#L50) component, line 50 fails since `formatBytes(parseInt(undefined))` fails.
